### DISCUSSION TITLE
Saving pipelines

### DISCUSSION THREAD

**Discussion by Hboni** [Hboni/francis#12] [Ideas]
I think that being able to save pipelines can be really useful! For now, I did not look a lot at your code, I don't know how everything is organized, how processes are stored.
How do you think that we can save pipeline? Using JSON?

Also, this discussion is to structure this feature, but @jeaunrg do you think that it will be in the first "release", or you think we must keep it for later?

**Comment by jeaunrg** [comment]
The architecture of the graph is really simple: we only have to store the name of each node and their parent names in a dictionnary.
So for now the graph can be saved in config directory using JSON like you said.

The save/restore functions are already committed (for the graph architecture only, not settings), you can try it now with ctrl+S / ctrl+R.
But I agree we must keep it for later...
